lib/jekyll/plugin.rb

```diff
@@ -60,10 +60,7 @@ def self.priority(priority = nil)
     #
     # Returns the safety Boolean.
     def self.safe(safe = nil)
-      if safe
-        @safe = safe
-      end
-      @safe || false
+      @safe ||= !!safe
```

Comment:
You're right i tried to change the code now similar to the `_suffix` / `_prefix` stuff because i think it is closest to the method description in this case as well.